app/controllers/orders_controller.rb

```diff
@@ -1,9 +1,19 @@
 class OrdersController < ApplicationController
   def index
     @orders = current_user.orders
+    # @user.orders.preload(:items)
   end
 
   def show
     @order = Order.find(params[:id])
   end
+
+  def new
+		order = Order.create(status: "ordered", user_id: current_user.id)
+		item_hash = @cart.order_hash
+		order.add(item_hash)
+		@cart.destroy
+		flash[:success] = "Order was successfully placed"
```

Comment:
lets ask illana if we can make up our own messages that are more suitable to our site